divi/src/Makefile.am

```diff
@@ -584,7 +584,7 @@ divid_LDADD += $(LIBBITCOIN_ZMQ) $(ZMQ_LIBS)
 endif
 
 if ENABLE_WALLET
-divid_LDADD += libbitcoin_wallet.a \
+divid_LDADD += $(LIBBITCOIN_WALLET) \
   $(LIBBITCOIN_CRYPTO)
```

Comment:
We  could pull these makefile changes into their own separate PR since it's clear they're separate from the PR.

Reply:
Good point, I've split this out (and extended it) in #64.  The reason why this was part of this PR is that it needed more extensive changes at a previous state of the PR to make it build, and this part (which is indeed not specific to the current PR) remained.